src/c++/perf_analyzer/concurrency_worker.h

```diff
@@ -139,6 +139,53 @@ class ConcurrencyWorker : public LoadWorker {
   // Detect and handle the case where this thread needs to exit
   // Returns true if an exit condition was met
   bool HandleExitConditions();
+
+  void WaitForOngoingRequests();
+
+  void SyncClientStats()
```

Comment:
Should these definitions be moved to the `.cc` file?

Reply:
It is sufficiently small that its probably okay to leave

Reply:
Up to you guys. I'm fine with small functions in the .h. I'm also fine putting everything in .cc

Reply:
I moved them.